aviatrix/resource_aviatrix_saml_endpoint.go

```diff
@@ -43,6 +43,13 @@ func resourceAviatrixSamlEndpoint() *schema.Resource {
 				ForceNew:    true,
 				Description: "Custom Entity ID. Required to be non-empty for 'Custom' Entity ID type, empty for 'Hostname'.",
 			},
+			"msg_template": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Default:     "",
```

Comment:
Not sure if default should be empty. It might cause a diff. Maybe you should use the default template on the controller as default